test/gtest/ucp/ucp_test.cc

```diff
@@ -228,6 +228,8 @@ ucs_status_t ucp_test::request_process(void *req, int worker_index, bool wait)
             progress(worker_index);
             status = ucp_request_check_status(req);
         } while ((status == UCS_INPROGRESS) && (ucs_get_time() < deadline));
+        EXPECT_NE(UCS_INPROGRESS, status) << "request " << req
+                                          << " was not completed on time";
```

Comment:
duplication of error in line 239?

Reply:
👍 probably rebase issue, I'd remove that one, here it looks simpler